src/components/NoteRoot/NoteRoot.js

```diff
@@ -68,10 +69,14 @@ class NoteRoot extends React.Component {
 
   render() {
     const { annotation, renderContents, isEditing, closeEditing, searchInput, contents } = this.props;
+    const annotationState = annotation.getStatus();
 
     return (
       <div className="NoteRoot">
         {this.renderHeader()}
+        {annotationState && annotationState !== '' && annotationState !== 'None' &&
+          <div className="status">Status: {annotationState}</div>
```

Comment:
Do we also want to translate the `Status` word?

Reply:
Resolved in dcb321c. Thank you!